invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json

```diff
@@ -13,8 +13,7 @@
   },
   "affiliations": {
     "type": "array",
-    "items": {"$ref": "#/affiliation"},
-    "uniqueItems": true
+    "items": {"$ref": "#/affiliation"}
```

Comment:
I am not sure we are following semantic versioning on this. It looks like not fully, see here history: https://github.com/inveniosoftware/invenio-rdm-records/commits/master/invenio_rdm_records/records/jsonschemas/records/definitions-v2.0.0.json

If you think I shold create a new one, let me know and I will do it.